src/zenml/cli/stack_components.py

```diff
@@ -122,14 +175,18 @@ def describe_stack_component_command(name: Optional[str]) -> None:
                 )
             return
 
-        try:
-            active_component_name = repo.active_stack.components[
-                component_type
-            ].name
-            is_active = active_component_name == component.name
-        except KeyError:
-            # there is no component of this type in the active stack
-            is_active = False
+        is_active = False
+        if active_stack_name:
+            try:
+                active_component_name = (
+                    repo.get_stack(active_stack_name)
```

Comment:
Does `repo.active_stack` no longer work? How about making it work again by putting all of this logic from here into that property?

Reply:
Same here, I removed it. But yes, `repo.active_stack` still works fine.